package.json

```diff
@@ -15,18 +15,6 @@
   },
   "author": "Adalo",
   "license": "ISC",
-  "adalo": {
```

Comment:
if you remove the adalo property here, you need to add the adalo.json file

Reply:
Whoops, npx (?!) removed it from package.json and I missed the adalo.json file. Added!